service/orchestrator/orchestrator.go

```diff
@@ -92,15 +92,15 @@
 
 		b, err := ioutil.ReadFile(fileName)
 		if err != nil {
-			log.Errorf("Could not retrieve default configuration for metric %s: %s. Ignoring metric", m.Id, err)
+			log.Errorf("Could not retrieve default configuration for metric %s: %v. Ignoring metric", m.Id, err)
```

Comment:
`%s` would also work since interface method `Error()` is of type string. But %v maybe is the common usage.

Reply:
Now I am confused. Do you want to use `%v` or `%w` for `log.Errorf`? As I understood, the common way is to use `%v` for errors.